dask/tests/test_order.py

```diff
@@ -589,7 +664,13 @@ def test_dont_run_all_dependents_too_early(abcde):
         dsk[(c, i)] = (f, (c, 0))
         dsk[(d, i)] = (f, (d, i - 1), (b, i), (c, i))
     o = order(dsk)
-
+    # The expected numbers here stand for the central computation branch.
+    # Ideally, they are exactly three apart since we only want to load two nodes
+    # before moving forward with a central node.
+    # This pattern is currently broken for the next to last node since we're
+    # eagerly freeing up the root node before processing once there are no
+    # further dangling dependents. Earlier versions of dask.order behaved
+    # different but this is a net-zero memory pressure operation so it is fine.
```

Comment:
It looks to me like expected is spaced equally. What exactly is broken here?

Reply:
this is outdated